app/assets/javascripts/restfulTypeahead.js

```diff
@@ -17,29 +17,25 @@ $(function() {
       source: debounce(function (query, process) {
 
         query = query.replace(' ', '-'); // replace spaces with hyphens
-        var encoded_query = encodeURIComponent(query); 
-        var qryType = $(el).attr('qryType'); 
-        if (qryType == "tags") {
```

Comment:
This looks good so far @ghostoverflow, I think you mistakenly removed line 22... it wasn't part of the issue changes...

Reply:
@RuthNjeri @ghostoverflow Actually, I think this change was intentional! Someone else made a PR for this that was failing because of codeclimate. See https://github.com/publiclab/plots2/pull/9190#issuecomment-778729539

@ghostoverflow was just taking those suggestions, and it looks like they pass. It also looks like you fixed some whitespace issues. Thanks!

Reply:
Going to resolve this conversation if that's okay!

Reply:
I only changed the things that are required and other changes are because of my IDE. It automatically formats the file in it's own way. I'm new to PR-things and gradually learning to work on GitHub, if there is something wrong I'll be happy to work on it.

Reply:
Thanks for the clarification @noi5e